payment-stellar/src/main/java/org/stellar/anchor/paymentservice/stellar/StellarPaymentsService.java

```diff
@@ -0,0 +1,170 @@
+package org.stellar.anchor.paymentservice.stellar;
+
+import com.google.gson.Gson;
+import java.math.BigDecimal;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import org.stellar.anchor.exception.HttpException;
+import org.stellar.anchor.paymentservice.Account;
+import org.stellar.anchor.paymentservice.DepositInstructions;
+import org.stellar.anchor.paymentservice.DepositRequirements;
+import org.stellar.anchor.paymentservice.Network;
+import org.stellar.anchor.paymentservice.Payment;
+import org.stellar.anchor.paymentservice.PaymentHistory;
+import org.stellar.anchor.paymentservice.PaymentsService;
+import org.stellar.anchor.paymentservice.stellar.requests.SubmitTransactionRequest;
+import org.stellar.anchor.paymentservice.stellar.util.NettyHttpClient;
+import org.stellar.sdk.*;
+import org.stellar.sdk.responses.AccountResponse;
+import org.stellar.sdk.responses.SubmitTransactionResponse;
+import reactor.core.publisher.Mono;
+import reactor.netty.ByteBufMono;
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.http.client.HttpClientResponse;
+
+@Data
```

Comment:
Marking CirclePaymentService with lombok @Data is anti-pattern. (it's a service, not a data) 

Marking a class with @Data is like a data class in Kotlin.

I suggest that we add `getNetwork`, `getUrl` and `getSecretKey` and pass the values to constructors.

Reply:
Oh ok, I was just following the pattern set in the other projects. I'll fix this up.

Reply:
No problem. I was discussing with Marcelo about the CirclePaymentService. And he reminded me that you have the same annotation.